monitoring/src/monitoring_service/states.py

```diff
@@ -0,0 +1,117 @@
+from dataclasses import dataclass, field  # isort:skip noqa differences between python 3.6 and 3.7
+from typing import List, Optional
+
+from eth_utils import decode_hex, to_checksum_address
+
+from raiden_contracts.constants import ChannelState, MessageTypeId
+from raiden_libs.utils import eth_recover, pack_data
+
+
+@dataclass
+class Channel:
+    token_network_address: str
+    identifier: int
+    participant1: str
+    participant2: str
+    settle_timeout: int
+    state: ChannelState = ChannelState.OPENED
+    closing_block: Optional[int] = None
+
+
+@dataclass
+class MonitoringServiceState:
+    token_network_registry_address: str
+    monitor_contract_address: str
+    latest_known_block: int
+    token_network_addresses: List[str] = field(default_factory=list)
+
+
+@dataclass
+class MonitorRequest:
+    # balance proof
+    channel_identifier: int
+    token_network_address: str
+    chain_id: int
+
+    balance_hash: str
+    nonce: int
+    additional_hash: str
+    signature: str
+
+    # FIXME: necessary?
```

Comment:
If that's properly baked into the signing function (and doesn't change), it's not needed.

Reply:
@Dominik1999 These seem to be unnecessary to send with the message. I removed them here.